components/movementsensor/movementsensor.go

```diff
@@ -74,7 +78,7 @@ type MovementSensor interface {
 	CompassHeading(ctx context.Context, extra map[string]interface{}) (float64, error) // [0->360)
 	Orientation(ctx context.Context, extra map[string]interface{}) (spatialmath.Orientation, error)
 	Properties(ctx context.Context, extra map[string]interface{}) (*Properties, error)
-	Accuracy(ctx context.Context, extra map[string]interface{}) (map[string]float32, error)
+	Accuracy(ctx context.Context, extra map[string]interface{}) (map[string]float32, float32, float32, NmeaGGAFixType, float32, error)
```

Comment:
Maybe we could do a message returned

With a structure like:
```go
Accuracies {
     Accuracy map[string]interface{}
     HDOP float64
     VDOP float64
     NmeaGGAFix int
     CompassError
}
```
And then you can call them from the return of that function.
But ask around.
```

Reply:
Actually hold off on this suggestion. We were asked to implement it this way with five returns so let me double check.

Reply:
Just double checked, we should try to wrap the five returns in a structure, and return that structure from a call to this API.

Reply:
Got it!